message/injector.cpp

```diff
@@ -1581,6 +1581,13 @@ void Injector::insertMessages()
 
         // Now we insert the headers and bodies of every MIME bodypart.
 
+        Bodypart *bp;
+        if ( m->hasPGPsignedPart() ) {
+            EString pnr( "raw-pgp-signed" );
```

Comment:
It may be a grandchild or a great-grandchild of the multipart/signed.
Example in attached signed_tree.txt .